pom.xml

```diff
@@ -134,7 +134,9 @@
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/pipeline-model-definition-plugin</gitHubRepo>
     <jenkins.baseline>2.479</jenkins.baseline>
-    <jenkins.version>${jenkins.baseline}.1</jenkins.version>
+    <jenkins.version>${jenkins.baseline}.3</jenkins.version>
```

Comment:
Forced by https://github.com/jenkinsci/analysis-pom-plugin/pull/1073#issuecomment-2679212385, pending bump to 2.492.1.